### DISCUSSION TITLE
Use conventional keyboard shortcuts

### DISCUSSION THREAD

**Discussion by KingSupernova31** [godotengine/godot-proposals#11075] [Editor]
There are several keyboard shortcuts that are conventional across many programs, but Godot does not support them. Some that I've come across on MacOS:

* Command+up/down: Scroll to top/bottom of page (works in GDscript editor, but not the documentation)
* Command+shift+up/down: select all text above/below
* Command+option+left/right: Go to left/right tab up/down script
* Command+backtick: change windows

Attempting to use these shortcuts in Godot often leads to totally different behavior than expected. This is confusing and unnecessary. Godot should use normal shortcuts for normal tasks, and only deviate from accepted practice when there is some specific need.

**Comment by Calinou** [comment]
Are these macOS-specific shortcuts? macOS has different handling of <kbd>Home</kbd>/<kbd>End</kbd> and <kbd>PgUp</kbd>/<kbd>PgDn</kbd> compared to other operating systems, specifically for multiline text.

> Command/Control+up/down: Scroll to top/bottom of page (works in GDscript editor, not documentation)

<kbd>Up</kbd>/<kbd>Down</kbd> arrows will scroll only by a single line. Do you mean <kbd>Home</kbd>/<kbd>End</kbd> or <kbd>PgUp</kbd>/<kbd>PgDn</kbd> instead?

**Reply by KingSupernova31** [reply]
Yes, on Mac, sorry. Edited to specify.

Of course if there are inconsistent shortcuts on Windows, those should be fixed too.
